README.md

```diff
@@ -3,3 +3,46 @@
 Сайт [rename.kr.ua](http://rename.kr.ua/) створено, щоб допомогти жителям міста знайти як саме змінилися назви вулиць, згідно с законом про декомунізаціюю
 
 Information about renamed streets in Kirovohrad, Ukraine in 2015-2016
```

Comment:
Also I think, readme file should be provided by using only one language, for example `ukrainian`.

Reply:
To put it mildly - yes...